routing.rst

```diff
@@ -2002,7 +2002,8 @@ avoids the need for duplicating routes, which also reduces the potential bugs:
         {
             #[Route(path: [
                 'en' => '/about-us',
-                'nl' => '/over-ons'
+                'nl' => '/over-ons',
+                '/about-us'                // Default path for undefined locales
```

Comment:
Is that available only with attribute ? I think we should keep same examples for attribute/yaml/xml/php if possible